packages/react/src/types/transactions.ts

```diff
@@ -1,12 +1,13 @@
-import { BigNumber } from "@ethersproject/bignumber";
+import type { Hex } from "viem";
+import type { Address } from "wagmi";
```

Comment:
We can merge the imports.

Reply:
Addressed in c3c8237483628f7281ec31eaf824da90db2b978b